README.md

```diff
@@ -129,7 +130,10 @@ I would like to thank my coding partners that helped me alot for this project.
 
```

Comment:
![image](https://github.com/Mahdi-Niazi/Vet-Clinic-Database/assets/93039359/02ba2451-d27f-4e80-bd34-4090df7cae12)

- [ ] It would be nice if you fix your issues page link because it is broken 😨 Kindly fix it to make your readme descriptive and professional 😊